Nim/Nim/Nim.cpp

```diff
@@ -0,0 +1,133 @@
+using namespace std;
+
+#include <iostream>
+#include <string>
+#include <cmath>
+
+int MatchSelect(int Num, int Total, string PlayerName) // I always get the total matches left for other player from here.
```

Comment:
Remember to name function parameters with lowerCase `num`, `total`, `playerName`